packages/ui/src/components/DatePicker/Calendar/Calendar.tsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react'
-import { Box, Stack, Text } from '@highlight-run/ui'
+import { Box } from '../../Box/Box'
+import { Stack } from '../../Stack/Stack'
+import { Text } from '../../Text/Text'
```

Comment:
Had to do a bunch of untangling of circular dependencies here.

Reflame's development mode makes use of [es-module-shims](https://github.com/guybedford/es-module-shims)' shim mode to be able to hook into module loading and update import maps at runtime. It generally does a good job of supporting circular imports transparently, but it seems to run into some trouble with this specific pattern where index.ts is full of reexports. I'll try to find a minimal repro and file an issue there.